src/components/ui/Button/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import {
+  Button as RNButton,
+  ButtonProps as RNButtonProps,
+} from "react-native-elements";
+
+interface ButtonInputProps extends RNButtonProps {
+  id: string;
+}
+
+const Input: React.FC<ButtonInputProps> = ({ id, ...rest }) => (
+  <RNButton data-testid={id} {...rest}/>);
```

Comment:
Might be good to add default styles (like stretching to the full width of the screen and padding)

Reply:
Oh and also the text needs to be in montserrat font. react-native-elements put everything in Roboto by default I believe

Reply:
think padding on buttons should default to 0 but we can use the padding with containers

Reply:
just saw second comment :) Montserrat is good, we will also need to add the font pairing for Nokora font. It's used in all main headings. 
Added the dependency but wasn't sure how to load it so will come back to that